providers/keap/parse.go

```diff
@@ -15,3 +17,70 @@ func makeNextRecordsURL(moduleID common.ModuleID) common.NextPageFunc {
 		return jsonquery.New(node).StrWithDefault("next_page_token", "")
 	}
 }
+
+// Before parsing the records if applicable API will be called one time to get model describing custom fields.
```

Comment:
```suggestion
// Before parsing the records, if any custom fields are present (without a human readable name), this will call the correct API to extend & replace the custom field with human readable information.
```